stub-idp/src/main/java/uk/gov/ida/stub/idp/resources/idp/RegistrationPageResource.java

```diff
@@ -51,34 +58,40 @@
 
 @Path(Urls.REGISTER_RESOURCE)
 @Produces(MediaType.TEXT_HTML)
-@SessionCookieValueMustExistAsASession
+//@SessionCookieValueMustExistAsASession
```

Comment:
We shouldn't leave commented code in ... however, see next comment ...

Reply:
I spotted that and took action already ...